application/pages/modplatform/technic/TechnicPage.cpp

```diff
@@ -85,86 +86,86 @@ void TechnicPage::onSelectionChanged(QModelIndex first, QModelIndex second)
 
 void TechnicPage::suggestCurrent()
 {
-    if (!isOpened)
-    {
-        return;
-    }
-    if (current.broken)
-    {
-        dialog->setSuggestedPack();
-        return;
-    }
-
-    QString editedLogoName;
-    editedLogoName = "technic_" + current.logoName.section(".", 0, 0);
-    model->getLogo(current.logoName, current.logoUrl, [this, editedLogoName](QString logo)
-    {
-        dialog->setSuggestedIconFromFile(logo, editedLogoName);
-    });
-
-    if (current.metadataLoaded)
-    {
-        metadataLoaded();
-    }
-    else
+    if (isOpened)
     {
-        NetJob *netJob = new NetJob(QString("Technic::PackMeta(%1)").arg(current.name));
-        std::shared_ptr<QByteArray> response = std::make_shared<QByteArray>();
-        QString slug = current.slug;
-        netJob->addNetAction(Net::Download::makeByteArray(QString("https://api.technicpack.net/modpack/%1?build=multimc").arg(slug), response.get()));
-        QObject::connect(netJob, &NetJob::succeeded, this, [this, response, slug]
+        if (!current.broken)
```

Comment:
This sort of rewrite makes the code actually worse.
Early exit and less levels of indentation = more readable, easier to refactor code.

So I consider this change to be a regression (on top of being basically impossible to review because of the large structural changes).